src/mame/drivers/nes_vt.cpp

```diff
@@ -1417,6 +1528,32 @@ void nes_vt_state::nes_vt_base(machine_config &config)
 	m_apu->add_route(ALL_OUTPUTS, "mono", 0.50);
 }
 
+void nes_vt_ablpinb_state::nes_vt_ablpinb(machine_config &config)
+{
+	nes_vt_base(config);
+
+	if (0)	// the dumped ABL Pinball unit is PAL (26.6017 XTAL) although Asian ones are likely NTSC.  PAL NES emulation seems more broken at the moment, probably timing issues in the PPU
```

Comment:
Why would Asian ones be NTSC?  Most of Asia uses PAL (besides Japan with NTSC and some SECAM holdouts).

Reply:
ah ok, I thought it was an NTSC region as I'm used to seeing NTSC-C for China, although that appears to be a 2003+ thing.  some of the Asian units we've seen have had NTSC clocks tho.

Reply:
Sony regions are weird.  NTSC-C what Sony calls the copy protection region for PRC excluding Hong Kong and Macau.  The consoles output PAL video.  Sony includes Hong Kong and Macau in the NTSC-J region for copy protection purposes, although TVs in these territories also use PAL standard.

Reply:
anyway, i'll just set it to the PAL clone timings as mixmaster put forward as unlike plain PAL this seems to leave it in a stable enough state to try and figure out what they've done with the inputs, which is very much non-standard here.